### PR TITLE
ISM feedback

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/ism/managedindices.md [hunk 1]
```diff
@@ -66,7 +64,7 @@ If an index is stuck in its current state, never proceeding, and you want to upd
 
 If you update the policy without including an identical state, ISM updates the policy only after all actions in the current state finish executing. Alternately, you can choose a specific state in your old policy after which you want the new policy to take effect.
 
-To change a policy, do the following:
+To change a policy using Kibana UI, do the following:
```
Comment (aetter) [inline docs/ism/managedindices.md]: either "Kibana" (preferred) or "the Kibana UI"

File: docs/ism/policies.md [hunk 2]
```diff
@@ -60,24 +60,27 @@ This table lists the parameters that you can define for an action.
...
 :--- | :--- |:--- |:--- |
-`timeout` |  The timeout period for the action. Accepts time units for minutes, hours, and days. | `time unit` | No | Specific to action
+`timeout` |  The timeout period for the action. Accepts time units for minutes, hours, and days. | `time unit` | No | -
 `retry` | The retry configuration for the action. | `object` | No | Specific to action
-`count` | The number of retry counts. | `number` | No | Specific to action
-`backoff` | The backoff policy type to use when retrying. | `string` | No | Exponential
-`delay` | The time to wait between retries. Accepts time units for minutes, hours, and days. | `time unit` | No | Specific to action
 
-All actions have common `timeout`, `retry`, and `backoff` settings. If you don't specify these settings, ISM uses its defaults.
+The `retry` operation has the following parameters:
```
Comment (aetter) [inline docs/ism/policies.md]: Are we comfortable calling these parameters rather than fields?

Comment (ashwinkumar12345) [reply docs/ism/policies.md]: I have used "parameters" when the user can specify them as inputs and "fields" when some of them are read only and used as a reference. Is that ok?

Comment (aetter) [reply docs/ism/policies.md]: Yep, that's fine. Just wanted to make sure there was a reason and it wasn't an oversight.

**Pull request merged**